CMakeLists.txt

```
@@ -334,6 +336,72 @@ if(URIPARSER_BUILD_TESTS)
     add_custom_target(check COMMAND ${CMAKE_CTEST_COMMAND})
 endif()
 
+#
+# Fuzzers
+#
+if(URIPARSER_BUILD_FUZZERS)
+    if(NOT "${CMAKE_C_COMPILER_ID}" STREQUAL "Clang")
+        message(FATAL_ERROR
+            "Building fuzz targets without Clang (but ${CMAKE_C_COMPILER_ID}) is not "
+            "supported. Please set -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++.")
+    endif()
+
+    if(URIPARSER_OSSFUZZ_BUILD AND NOT DEFINED ENV{LIB_FUZZING_ENGINE})
+        message(FATAL_ERROR
+            "OSS-Fuzz builds require the environment variable "
+            "LIB_FUZZING_ENGINE to be set. If you are seeing this "
+            "warning, it points to a deeper problem in the ossfuzz "
```

```suggestion
            "warning, it points to a deeper problem in the OSS-Fuzz "
```
(I should fix this in Expat, too.)

---

Corrected